### PR TITLE
[engine] disable affinity APIs on QC chips due to broken linux kernel.

### PR DESCRIPTION
**Pull request by jonahwilliams**
Disable affinity on qcom chips. FYI @johnmccutchan . We haven't recieved a direct bug report, but someone reached out via the android team and warned us about this. 🤷‍♂️

### CODE REVIEW

File: fml/platform/android/cpu_affinity.cc [hunk 1]
```diff
@@ -20,6 +22,9 @@ namespace fml {
 std::once_flag gCPUTrackerFlag;
 static CPUSpeedTracker* gCPUTracker;
 
+std::once_flag gCheckManufacturerFlag;
```
Comment (chinmaygarde) [inline fml/platform/android/cpu_affinity.cc:25]: Nit: You should be able to put both these as statics in the `SetUpCPUTracker` method without polluting the namespace.

**Review by chinmaygarde — APPROVED** [review]

**Comment by johnmccutchan** [comment]
Can this be closed?

**Comment by jonahwilliams** [comment]
Depends on what you want to do here. I'm happy either way.

**Comment by johnmccutchan** [comment]
My vote is do nothing.

**Comment by jonahwilliams** [comment]
SGTM

**Pull request closed without merging**